assets/js/views/ChargingSessions.vue

```diff
@@ -228,6 +226,16 @@ export default {
 			const modal = Modal.getOrCreateInstance(document.getElementById("sessionDetailsModal"));
 			modal.show();
 		},
+		csvHrefLink(locale, month) {
+			var url = "./api/sessions?format=csv&lang=" + locale;
+			if (month) {
+				url += "&year=" + month.split(".")[0];
```

Comment:
Mach hier lieber vorher einmal ein Desctructuring (siehe oben `const [year, month] = group.split(".");`). Dann braucht man nicht mehrfach splitten und der Code ist einfacher zu verstehen.

Generell bevorzugen wir in der Codebasis Templatestrings gegenüber `+` beim Stringzusammenbau. Gibt bestimmt ne Eslint Regel, die wir dafür mal anschalten sollten. Dann ist das klarer und vor allem Einheitlich.